lab-spencer/test/server-test.js

```diff
@@ -0,0 +1,152 @@
+'use strict';
+
+const superagent = require('superagent');
+const expect = require('expect');
+const server = require('../lib/server.js');
+const Seahawk = require('../model/seahawk.js');
+const PORT = process.env.PORT || 3000;
+
+describe('Seahawk Constructor', () => {
+  it('Should return a Seahawk Object', () => {
+    let me = new Seahawk(1, 'Spencer Gietzen', '5\'10"', '150', 'TB', 'rand/pic.png');
+    expect(me.id).toEqual(1);
+    expect(me.name).toEqual('Spencer Gietzen');
+    expect(me.height).toEqual('5\'10"');
+    expect(me.weight).toEqual('150');
+    expect(me.position).toEqual('TB');
+    expect(me.picture).toEqual('rand/pic.png');
+  });
+});
+
+describe('/api/seahawks routes', () => {
+  let tempSeahawk;
+  before(done => {
+    server.listen(PORT, () => done());
+  });
+  after(done => {
+    server.close(() => done());
+  });
+  describe('POST', () => {
+    it('Should respond 201 with stringified JSON of the player posted', done => {
+      superagent.post(`localhost:${PORT}/api/seahawks`)
+        .send(JSON.stringify({name: 'Russell Wilson', height: '6\'11"', weight: '500', position: 'QB', picture: 'testpic/pic.png'}))
+        .end((err, res) => {
+          if (err) return done(err);
+          expect(res.status).toEqual(201);
+          expect(res.body.id).toExist();
```

Comment:
Tip, you can compare entire objects with `.toEqual`. In this case since the ID will be generated, it you will have to work around it.

```js
expect(res.body).toEqual({
  name: 'Russell Wilson',
  height: '6\'11"',
  weight: '500',
  position: 'QB',
  picture: 'testpic/pic.png',
});
```